### PR TITLE
backport 'Add systemctl daemon-reload handler to openshift_node' #4403 to openshift_node_upgrade

### PR DESCRIPTION
**Pull request by ingvagabund**
Cause we want both openshift_node and openshift_node_upgrade to be synced

### CODE REVIEW

**Comment by ingvagabund** [comment]
aos-ci-test

File: roles/openshift_node_upgrade/handlers/main.yml [hunk 1]
```diff
@@ -29,3 +29,6 @@
   when:
   - (not skip_node_svc_handlers | default(False) | bool)
   - not (node_service_status_changed | default(false) | bool)
+
+- name: reload systemd units
```
Comment (kwoodson) [inline roles/openshift_node_upgrade/handlers/main.yml]: Can we add a note here to use the ansible module when it is supported or use it if it is already supported?
http://docs.ansible.com/ansible/latest/systemd_module.html

Comment (ingvagabund) [inline roles/openshift_node_upgrade/handlers/main.yml]: Added that as a TODO item.

**Comment by ingvagabund** [comment]
aos-ci-test

**Comment by ingvagabund** [comment]
[merge]

**Pull request merged**